src/services/files/settings/index.ts

```diff
@@ -0,0 +1,19 @@
+import { HttpClient } from '../../http-client';
+import { HttpInstance } from '../../../http/types';
+import { FileServiceSettingsService, FileServiceSettingsUpdate } from './types';
+import { OptionsBase } from '../../types';
+
+export default (
+  client: HttpClient,
+  httpAuth: HttpInstance
+): FileServiceSettingsService => ({
+  async get(options: OptionsBase) {
+    const { data } = await client.get(httpAuth, `/settings`, options);
+    return data;
+  },
+
+  async update(data: FileServiceSettingsUpdate, options: OptionsBase) {
+    const response = await client.put(httpAuth, `/settings`, data, options);
+    return response.data;
```

Comment:
Very minor: but the destructuring is in consistent between both functions

Reply:
For the put requests we call the input `data` for consistency meaning that we can destructure the output to data.

Any better suggestions? 🤔